src/openforms/scss/components/admin/_logic-action.scss

```diff
@@ -25,4 +45,10 @@ $logic-line-breakpoint: 1800px !default;
     padding-top: 7px;
     flex-grow: 0;
   }
+
+  &__errors {
+    max-height: $error-height;
```

Comment:
Ah, so this would introduce a scrollbar probably if there's more content :thinking: